### DISCUSSION TITLE
YOLO (YOU ONLY LIVE ONCE?)

### DISCUSSION THREAD

**Discussion by kikiru328** [kikiru328/Bone_Detection#54] [Announcements]
# 어림도 없지
* Box나 쳐라! 
## 홀수만 쳐라!

* 승혜                 
F >3Y - 8Y        
* 혜정
F > 9Y - 14Y 
* 광훈
F > 15Y - M> 6Y
* 유빈                       
M >7Y - 12Y          
* 종현
M > 13Y - 17Y + tjnet
# 결과는 공유폴더 train_data 폴더에 넣어라
![image](https://user-images.githubusercontent.com/60537388/139405508-9f45c369-bb47-4d0e-aa83-1531db6bdd2f.png)
![image](https://user-images.githubusercontent.com/60537388/139405465-406fd1e3-79df-4727-b750-7e1f8f1b97b4.png)
![image](https://user-images.githubusercontent.com/60537388/139405668-525c5d41-1915-42af-8100-661e241a8141.png)

**Comment by Jjongu** [comment]
무지성 박스 멈춰!
![image](https://user-images.githubusercontent.com/92709187/139400433-64307adc-9928-4ee2-9b6d-4eaf2c04360e.png)

**Comment by kikiru328** [comment]
![image](https://user-images.githubusercontent.com/60537388/139522454-25ce34fa-2e36-4ca8-971b-3da055c815c8.png)

**Reply by hye-jj** [reply]
공지 감사합니다!

**Comment by hye-jj** [comment]
광훈오빠가 공유해준 블로그 : https://lynnshin.tistory.com/48?category=941325

## mAP, IoU 설명
### 1) Intersection over union(IoU)
- 어떤 데이터셋에 대하여 객체 검출하는 모델의 정확도를 측정하는 평가지표.
- 합성곱 신경망을 사용한 객채 검출 모델(R-CNN, Faster R-CNN, YOLO 등)을 평가할 때 사용.
- 평가하기 위해서는 정답 바운딩 박스와 모델로부터 예측된 predicted bounding box 가 필요함.

![image](https://user-images.githubusercontent.com/86215536/139711285-82c08009-01aa-4848-8b1b-6236d5eb0ee6.png)
> 두 바운딩 박스를 가지고 IoU = 교집합 영역 넓이/합집합 영역 넓이 를 계산한 값.

### 2) mAP
- mAP@.5  => Pacal VOC의 mAP 평가 방식,  mAP@.5:.95 => COCO의 mAP 평가 방식
- Pacal VOC는 IoU > 0.5 인 detection을 True, 그 이하는 False로 평가하는 방식을 사용.
- COCO는 IoU > 0.5, IoU > 0.55, IoU > 0.6, ..., IoU > 0.95 각각을 기준으로 AP를 계산한 후 이들의 평균을 취하는 방식.

*AP : Average Precision, **precision-recall 그래프에서 그래프 선 아래쪽의 면적**으로 계산됨. 컴퓨터 비전 분야에서 물체 검출 및 이미지 분류 알고리즘의 성능은 대부분 이 AP로 평가함. 물체 클래스가 여러 개인 경우 각 클래스당 AP를 구한 다음에 그것을 모두 합한 다음에 물체 클래스의 갯수로 나눠줌으로 알고리즘의 성능을 평가 -> mAP(mean average precision), AP가 높을수록 그 알고리즘 성능이 전체적으로 우수하다는 의미.
